dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequestThreadData.java

```diff
@@ -40,13 +39,11 @@ public class SRTServletRequestThreadData {
     private LinkedList _queryStringList = null; // 256836
     private boolean _qsSetExplicit = false;
 
-
     private UnsynchronizedStack _paramStack = new UnsynchronizedStack();
```

Comment:
_paramStack field I believe can also be removed.

Reply:
Yup, done.